ReactAndroid/src/main/java/com/facebook/react/devsupport/WindowOverlayCompat.java

```diff
@@ -0,0 +1,18 @@
+package com.facebook.react.devsupport;
+
+import android.os.Build;
+import android.view.WindowManager;
+
+/**
+ * Compatibility wrapper for apps targeting API level 26 or later.
+ * See https://developer.android.com/about/versions/o/android-8.0-changes.html#cwt
+ */
+class WindowOverlayCompat {
```

Comment:
I think it could be useful to make this public, if you prefer not to let's just add a /* package */ comment like here for example https://github.com/AndrewJack/react-native/blob/f1cf378cf4ef3e5473fe2ace6d564be897e56a0e/ReactAndroid/src/main/java/com/facebook/react/devsupport/RedBoxDialog.java#L51

Reply:
I don't think it needs to be public. It's only needed in the `devsupport` package and I wouldn't like to encourage developers to use this either.

I'll add `/* package */`